pkg/common/host_utils.go

```diff
@@ -74,16 +76,19 @@ var ExecCommand = execCommandHelper
 // EnsureFreeLoopbackDeviceFile finds the next available loop device under /dev/loop*
 // If no free loop devices exist, a new one is created
 func EnsureFreeLoopbackDeviceFile() (uint64, error) {
-	LOOP_CTL_GET_FREE := uintptr(0x4C82)
-	LoopControlPath := "/dev/loop-control"
-	ctrl, err := os.OpenFile(LoopControlPath, os.O_RDWR, 0660)
+	loopControlPath := "/dev/loop-control"
+
+	// Open the loop-control device
+	ctrl, err := os.OpenFile(loopControlPath, os.O_RDWR, 0660)
 	if err != nil {
```

Comment:
It would be nice, at some point to consolidate all the copies of this function.
Not a priority for this pull request, though. Just something to consider for the future.